eng/pipelines/templates/steps/cosmos-emulator.yml

```diff
@@ -0,0 +1,17 @@
+parameters:
+  EmulatorMsiUrl: "https://aka.ms/cosmosdb-emulator"
+  StartParameters: ''
+
+steps:
+  - powershell: |
```

Comment:
@danieljurek  We should switch to pwsh instead of powershell as we are using pwsh (aka powershell 6) in all our other pipelines now.

//cc @chidozieononiwu

Reply:
Noted...

Reply:
LOL As an additional note you might need to stick with powershell see https://github.com/Azure/azure-sdk-for-java/pull/5633